dev-requirements.txt

```diff
@@ -1,3 +1,3 @@
 coverage
-six
```

Comment:
`six` is actually needed by the package, not just for testing. Moved to setup.py.